Dockerfile.mysql

```diff
@@ -2,10 +2,10 @@
 FROM mysql:latest
 
 # Environment variables for MySQL configuration
-ENV MYSQL_ROOT_PASSWORD=1234
-ENV MYSQL_DATABASE=app_db
-ENV MYSQL_USER=app_user
-ENV MYSQL_PASSWORD=1234
+ENV MYSQL_ROOT_PASSWORD="default"
```

Comment:
The environment variable `MYSQL_ROOT_PASSWORD` is set to 'default'. It's crucial to change this to a secure password for production use to ensure the security of your database.

Reply:
This is intended according to video lectures.